Engine_Revit_UI/Convert/Environment/ToBHoM/Panel.cs

```diff
@@ -348,8 +348,8 @@ public static oM.Environment.Elements.Panel ToBHoMEnvironmentPanel(this EnergyAn
         //        double aWidth = energyAnalysisOpening.Width;
         //        if (pullSettings.ConvertUnits)
         //        {
-        //            aHeight = UnitUtils.ConvertFromInternalUnits(aHeight, DisplayUnitType.DUT_METERS);
-        //            aWidth = UnitUtils.ConvertFromInternalUnits(aWidth, DisplayUnitType.DUT_METERS);
+                    //aHeight = aHeight.ToSI(UnitType.UT_Length);
```

Comment:
If this code is commented out and not needed perhaps let's remove it?

Reply:
This is BEnv part of the code, I prefer not to delete anything that is not mine, as long as I am not instructed to - would you like me to delete it?

Reply:
Yes please, let's try and be as tidy as we can :smile:

Reply:
Fixed.